main.go

```diff
@@ -1678,3 +1681,21 @@ func AddReleaseIntegrationSupport(ctx *pulumi.Context, cfg *config.Config, name
 
 	return nil
 }
+
+func AddHetznerHolochainInfraBucketsSecret(ctx *pulumi.Context, cfg *config.Config, repository string) error {
```

Comment:
Namespaced for holochain-infra project and using "buckets" plural because hetzner only lets you create s3 tokens for all buckets in a project.